object_database/web/cells/non_display/key_action.py

```diff
@@ -0,0 +1,41 @@
+"""KeyAction Cell
+
+Listens key-combo press events from the UI"""
+from ..cells import Cell
+
+
+class KeyAction(Cell):
+    """
+    Parameters
+    __________
+    keyCmd str: A string of a key combo like `Alt+i`, `Meta+D`
+                or event single keys like `X`.
+    callback func: A function that will be called with the
+                event response dictionary data as the sole arg.
+    wantedInfo list of str: A list of keys on the UI keydown event object
+                whose values should be sent back to the Cell in the response
+                message. Optional; defaults to None.
+    priority int: The priority level of the event response.
+    stopsPropagation bool: Whether or not this KeyAction should fire and then
+                stop other KeyActions with the same keyCmd from firing
```

Comment:
does this also prevent the key from propagating to other UX such as a code-editor?

Reply:
No. This is not possible directly, because Ace editor has it's own bizarre way of dealing with key events. What we can do, however, is start and stop our global KeyAction-based listener whenever a given Ace editor gains or loses focus. I'm working on that now and will push and update when it's up and running -- might be tomorrow the way things are going

Reply:
I think I have just pushed a solution. No need for a new test in Playground: just click into the CodeEditor and the keycombo (Alt+t) will stop working. Blur from it (click away) and it should start working again.